components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/Window.kt

```diff
@@ -6,46 +6,47 @@
 
 package mozilla.components.support.ktx.android.view
 
-import android.graphics.Color
 import android.os.Build
 import android.view.View
 import android.view.Window
 import androidx.annotation.ColorInt
-import mozilla.components.support.utils.ColorUtils.getReadableTextColor
+import mozilla.components.support.utils.ColorUtils.isDark
 
 /**
- * Colors the status bar to match the theme color.
+ * Colors the status bar.
  * If the color is light enough, a light status bar with dark icons will be used.
  */
-fun Window.setStatusBarTheme(@ColorInt themeColor: Int) {
+fun Window.setStatusBarTheme(@ColorInt color: Int) {
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
         val flags = decorView.systemUiVisibility
         decorView.systemUiVisibility =
-            flags.useLightFlag(themeColor, View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR)
+            flags.useLightFlag(color, View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR)
     }
-    statusBarColor = themeColor
+    statusBarColor = color
 }
 
 /**
- * Colors the navigation bar black or white depending on the background color.
+ * Colors the navigation bar.
+ * If the color is light enough, a light navigation bar with dark icons will be used.
  */
-fun Window.setNavigationBarTheme(@ColorInt backgroundColor: Int) {
+fun Window.setNavigationBarTheme(@ColorInt color: Int) {
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
         val flags = decorView.systemUiVisibility
         decorView.systemUiVisibility =
-            flags.useLightFlag(backgroundColor, View.SYSTEM_UI_FLAG_LIGHT_NAVIGATION_BAR)
+            flags.useLightFlag(color, View.SYSTEM_UI_FLAG_LIGHT_NAVIGATION_BAR)
     }
+    navigationBarColor = color
```

Comment:
This would set the navigation bar to any colour provided vs the light/dark colors you provide for the higher SDK. Do I understand that correct?

Reply:
Yes, it should set it to any colour to match with the custom tabs API.